<launch>

	<arg name="global_language" default="sv-SE"  />
	<arg name="global_condition" default="experiment"  />

	<node pkg="beginner_tutorials" name="fake_microphone" type="fake_microphone.py" output="screen" />
		<param name="language" type="string" value="$(arg global_language)" />

	<node pkg="beginner_tutorials" name="speakerDiarization" type="speakerDiarization.py" output="screen" />

	<node pkg="beginner_tutorials" name="wizardCommands" type="wizardCommands.py" output="screen" launch-prefix="gnome-terminal --command" />

	<node pkg="beginner_tutorials" name="guessComputation" type="guessComputation.py" output="screen" />
		<param name="language" type="string" value="$(arg global_language)" />

	<node pkg="beginner_tutorials" name="gameHandler" type="gameHandler.py" output="screen" />
		<param name="language" type="string" value="$(arg global_language)" />

	<node pkg="beginner_tutorials" name="furhatBridge" type="furhatBridge.py" output="screen" />
		<param name="language" type="string" value="$(arg global_language)" />
		<param name="robot_present" type="bool" value="true" />

	<node pkg="beginner_tutorials" name="behaviorModule" type="behaviorModule.py" output="screen" />
		<param name="language" type="string" value="$(arg global_language)" />
		<param name="condition" type="string" value="$(arg global_condition)" />
		<param name="robot_present" type="bool" value="false" />

</launch>